src/test/java/codeu/controller/NotificationServletTest.java

```diff
@@ -81,41 +79,127 @@ public void tesDoGet() throws IOException, ServletException {
     List<String> conversationInformation = new ArrayList<>();
     conversationInformation.add(fakeUser.getName());
     conversationInformation.add(fakeConversation.getTitle());
+    conversationInformation.add(fakeConversationId.toString());
     Event conversationEvent = new Event(UUID.randomUUID(), "Conversation", fakeConversation.getCreationTime(), conversationInformation);
     fakeEventList.add(conversationEvent);
 
+
     Message fakeMessage = new Message(UUID.randomUUID(), fakeConversationId, fakeUserId, "testMessage", Instant.ofEpochMilli(3000));
     List<String> messageInformation = new ArrayList<>();
     messageInformation.add(fakeUser.getName());
     messageInformation.add(fakeConversation.getTitle());
     messageInformation.add(fakeMessage.getContent());
+    messageInformation.add(fakeConversationId.toString());
     Event messageEvent = new Event(UUID.randomUUID(), "Message", fakeMessage.getCreationTime(), messageInformation);
     fakeEventList.add(messageEvent);
 
-    List<Event> spyEvents = Mockito.spy(eventsToShow);
-    spyEvents.add(conversationEvent);
-    spyEvents.add(messageEvent);
+    List<Event> eventsSinceLastSeenTime = new ArrayList<>();
+    eventsSinceLastSeenTime.add(conversationEvent);
+    eventsSinceLastSeenTime.add(messageEvent);
 
     Instant fakeLastSeenTime = Instant.ofEpochMilli(1500);
 
+    List<UUID> subbedId = new ArrayList<>();
+    subbedId.add(fakeConversationId);
+
     Mockito.when(mockSession.getAttribute("user")).thenReturn("fakeUser");
     Mockito.when(mockUserStore.getUser("fakeUser")).thenReturn(mockUser);
     Mockito.when(mockEventStore.getAllEvents()).thenReturn(fakeEventList);
     Mockito.when(mockUser.getLastSeenNotifications()).thenReturn(fakeLastSeenTime);
 
-    Mockito.when(mockEventStore.getEventsSince(fakeLastSeenTime)).thenReturn(spyEvents);
+    Mockito.when(mockEventStore.getEventsSince(fakeLastSeenTime)).thenReturn(eventsSinceLastSeenTime);
+    Mockito.when(mockUser.getSubscriptions()).thenReturn(subbedId);
 
     Instant currentTime = Instant.now();
     long nanosTosubtract = currentTime.getNano();
     Instant lastEventTime = currentTime.minusNanos(nanosTosubtract);
 
     NotificationServlet.doGet(mockRequest, mockResponse);
 
-    Mockito.verify(mockRequest).setAttribute("eventsToShow", spyEvents);
+    Mockito.verify(mockRequest).setAttribute("eventsToShow", eventsSinceLastSeenTime);
 
     Mockito.verify(mockUser).setLastSeenNotifications(lastEventTime);
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
 
   }
 
+  @Test
+  public void testDoGet_ifConversation() throws IOException, ServletException {
```

Comment:
Can we make the naming a little clearer for these two tests? I'm not sure what `ifConversation` tests.